From ebb6e620dd3aff13e898032af2e9494f8106e3d2 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Sat, 12 Jul 2014 17:02:19 -0400
Subject: [PATCH 2/2] build-sys: Add --disable-manpages option

Some systems won't have documentation build tools, or won't want docs
on the target system.  This option allows both to disable them.

Matches code from systemd.
---
 Makefile.am  | 2 ++
 configure.ac | 5 +++++
 2 files changed, 7 insertions(+)

diff --git a/Makefile.am b/Makefile.am
index 2f05ceb..eca194e 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -369,7 +369,9 @@ endif
 
 manual_pages = doc/btmon.1
 
+if ENABLE_MANPAGES
 dist_man_MANS += $(manual_pages)
+endif
 
 EXTRA_DIST += $(manual_pages:.1=.txt)
 
diff --git a/configure.ac b/configure.ac
index 9a7ab47..176eea8 100644
--- a/configure.ac
+++ b/configure.ac
@@ -174,6 +174,11 @@ if (test "${enable_client}" != "no"); then
 fi
 AM_CONDITIONAL(READLINE, test "${enable_readline}" = "yes")
 
+have_manpages=no
+AC_ARG_ENABLE(manpages, AS_HELP_STRING([--disable-manpages], [disable manpages]))
+AS_IF([test "x$enable_manpages" != xno], [have_manpages=yes])
+AM_CONDITIONAL(ENABLE_MANPAGES, [test "x$have_manpages" = "xyes"])
+
 AC_ARG_ENABLE(systemd, AC_HELP_STRING([--disable-systemd],
 		[disable systemd integration]), [enable_systemd=${enableval}])
 AM_CONDITIONAL(SYSTEMD, test "${enable_systemd}" != "no")
-- 
1.8.3.1

